integrated_channels/moodle/models.py

```diff
@@ -57,15 +57,6 @@ class MoodleEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
         )
     )
 
-    username = models.CharField(
-        max_length=255,
-        verbose_name="Webservice Username",
-        blank=True,
-        help_text=_(
-            "The API user's username used to obtain new tokens."
-        )
-    )
-
```

Comment:
As discussed, we should keep the 3 fields in the model for now

Reply:
@sameenfatima78 updated.